cmd/ledgerutil/main.go

```diff
@@ -42,29 +42,29 @@ func main() {
 	}
 
 	// Determine result json file location
-	var resultFilepath string
-	if outputDir != nil {
-		resultFilepath = *outputDir
-	} else {
-		resultFilepath, err = os.Getwd()
+	if outputDir == nil {
+		*outputDir, err = os.Getwd()
 		if err != nil {
-			fmt.Println(err)
+			fmt.Printf("%s%s\n", compareErrorMessage, err)
 			return
 		}
 	}
-	resultFilepath = filepath.Join(resultFilepath, resultFilename)
 
 	// Command logic
 	switch command {
 
 	case compare.FullCommand():
 
-		count, err := ledgerutil.Compare(*snapshotPath1, *snapshotPath2, resultFilepath)
+		count, err := ledgerutil.Compare(*snapshotPath1, *snapshotPath2, *outputDir)
 		if err != nil {
-			fmt.Println(err)
+			fmt.Printf("%s%s\n", compareErrorMessage, err)
 			return
 		}
-		fmt.Printf("\nSuccessfully compared snapshots. Result saved to %s. Total differences found: %d\n", resultFilepath, count)
+		fullpath, err := filepath.Abs(*outputDir)
+		if err != nil {
+			fmt.Printf("%s%s\n", compareErrorMessage, err)
+		}
```

Comment:
Had it print absolute path because it prints the literal input the user provides, so if they used the "." for current directory, that's what gets printed back. I'll have it print relative paths and just include edge cases for "." and "..".

Reply:
I think that it should be fine to print in the format user gave the input. IMO, lets not think too much in beautification of output at this stage.